dbos/_dbos_config.py

```diff
@@ -71,12 +173,53 @@ class ConfigFile(TypedDict, total=False):
     """
 
     name: str
-    language: str
     runtimeConfig: RuntimeConfig
     database: DatabaseConfig
     telemetry: Optional[TelemetryConfig]
     env: Dict[str, str]
-    application: Dict[str, Any]
+
+
+def is_config_file(obj: object) -> TypeGuard[ConfigFile]:
+    return (
+        isinstance(obj, dict)
+        and "name" in obj
+        and "runtimeConfig" in obj
+        and isinstance(obj.get("name"), str)
+        and isinstance(obj.get("runtimeConfig"), dict)
+    )
+
+
+def translate_dbos_config_to_config_file(config: DBOSConfig) -> ConfigFile:
+    db_config = parse_db_string_to_dbconfig(config["db_string"])
+    if "sys_db_name" in config:
+        db_config["sys_db_name"] = config.get("sys_db_name")
```

Comment:
This could still be `None`, but `_sys_db` and the like won't consider it if it doesn't evaluate to `True`